[2025-05-23T18:15:44.622-0600] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-23T18:15:44.671-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: libertyPr_health_score.send_to_elasticsearch scheduled__2025-05-23T00:00:00+00:00 [queued]>
[2025-05-23T18:15:44.689-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: libertyPr_health_score.send_to_elasticsearch scheduled__2025-05-23T00:00:00+00:00 [queued]>
[2025-05-23T18:15:44.689-0600] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-23T18:15:44.724-0600] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): send_to_elasticsearch> on 2025-05-23 00:00:00+00:00
[2025-05-23T18:15:44.736-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=5788) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-23T18:15:44.739-0600] {standard_task_runner.py:72} INFO - Started process 5790 to run task
[2025-05-23T18:15:44.742-0600] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'libertyPr_health_score', 'send_to_elasticsearch', 'scheduled__2025-05-23T00:00:00+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/health_score_lpr.py', '--cfg-path', '/tmp/tmpuoxkjlc4']
[2025-05-23T18:15:44.747-0600] {standard_task_runner.py:105} INFO - Job 56: Subtask send_to_elasticsearch
[2025-05-23T18:15:44.848-0600] {task_command.py:467} INFO - Running <TaskInstance: libertyPr_health_score.send_to_elasticsearch scheduled__2025-05-23T00:00:00+00:00 [running]> on host 439130e6349d
[2025-05-23T18:15:45.046-0600] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='libertyPr_health_score' AIRFLOW_CTX_TASK_ID='send_to_elasticsearch' AIRFLOW_CTX_EXECUTION_DATE='2025-05-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-23T00:00:00+00:00'
[2025-05-23T18:15:45.049-0600] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-23T18:15:45.106-0600] {logging_mixin.py:190} INFO - ðŸ“‹ Columns in ranking file: ['FIBER_NODE', 'CMTS_ID', 'Critical', 'Good', 'Offline', 'Warning', 'Total_Modems', 'Critical_Rate', 'Warning_Rate', 'Node_Size_Amplified', 'Health_Score', 'Attention_Priority']
[2025-05-23T18:15:47.612-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/elasticsearch/_sync/client/__init__.py:403: SecurityWarning: Connecting to 'https://10.212.56.57:9200' using TLS with verify_certs=False is insecure
  _transport = transport_class(

[2025-05-23T18:15:48.462-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T18:15:48.560-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.114s]
[2025-05-23T18:15:48.608-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T18:15:48.719-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.112s]
[2025-05-23T18:15:48.748-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T18:15:48.865-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.118s]
[2025-05-23T18:15:48.906-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T18:15:48.975-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.069s]
[2025-05-23T18:15:48.998-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T18:15:49.066-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.068s]
[2025-05-23T18:15:49.133-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T18:15:49.221-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.088s]
[2025-05-23T18:15:49.270-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T18:15:49.333-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.063s]
[2025-05-23T18:15:49.345-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T18:15:49.377-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.033s]
[2025-05-23T18:15:49.379-0600] {logging_mixin.py:190} INFO - âœ… Sent 3607 documents to Elasticsearch index 'hs_lpr'
[2025-05-23T18:15:49.381-0600] {python.py:240} INFO - Done. Returned value was: None
[2025-05-23T18:15:49.422-0600] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-23T18:15:49.423-0600] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=libertyPr_health_score, task_id=send_to_elasticsearch, run_id=scheduled__2025-05-23T00:00:00+00:00, execution_date=20250523T000000, start_date=20250524T001544, end_date=20250524T001549
[2025-05-23T18:15:49.557-0600] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-23T18:15:49.610-0600] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-23T18:15:49.613-0600] {local_task_job_runner.py:245} INFO - ::endgroup::
